pkg/bitmap/bitmap_test.go

```diff
@@ -0,0 +1,312 @@
+// Copyright 2021 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bitmap
+
+import (
+	"testing"
+)
+
+// generateFilledSlice generate a slice fill with numbers
+func generateFilledSlice(min, max, length int) []int32 {
+	if max == min {
+		return []int32{int32(min)}
+	}
+	if length > (max - min) {
+		return nil
+	}
+	randSlice := make([]int32, length)
+	if length != 0 {
+		rangeNum := int32((max - min)/length)
+		randSlice[0], randSlice[length-1] = int32(min), int32(max)
+		for i := 1 ; i < length - 1; i++ {
+			randSlice[i] = randSlice[i-1] + rangeNum
+		}
+	}
+	return randSlice
+}
+
+// compSlice return true if slice a equal to slice b
+func cmpSlice(a, b []int32) bool {
```

Comment:
Instead of writing this, you can just use `reflect.DeepEqual`.

Reply:
> Instead of writing this, you can just use `reflect.DeepEqual`.

I tried but `reflect.DeepEqual` just not works. Even though the two slices not equal, `reflect.DeepEqual` returns true.

Reply:
> Even though the two slices not equal, reflect.DeepEqual returns true.

I am not sure this is true. https://stackoverflow.com/a/15312182/6935187
I played with this in Golang playground, it seems to work. https://play.golang.org/p/nOh3Z2pO4f_k
Can you give a Playground example where it does not?

Reply:
> > Even though the two slices not equal, reflect.DeepEqual returns true.
> 
> I am not sure this is true. https://stackoverflow.com/a/15312182/6935187
> I played with this in Golang playground, it seems to work. https://play.golang.org/p/nOh3Z2pO4f_k
> Can you give a Playground example where it does not?

I will check it again. If `reflect.DeepEqual` still not works, I will show you the cases.
Also as this PR can not open for a strange reason, I will submit a new PR. We can discuss it there. Thanks for your check and patient.

Reply:
> > Even though the two slices not equal, reflect.DeepEqual returns true.
> 
> I am not sure this is true. https://stackoverflow.com/a/15312182/6935187
> I played with this in Golang playground, it seems to work. https://play.golang.org/p/nOh3Z2pO4f_k
> Can you give a Playground example where it does not?

`reflect.DeepEqual` works fine, I can not reproduce the failure, I have replaced `cmpSlice` with `reflect.DeepEqual`